app/helpers/markdown_tools_helper.rb

```diff
@@ -22,6 +22,7 @@ def md_button(name = nil, action: nil, label: nil, **attribs, &block)
     attribs.merge! href: 'javascript:void(0)',
                    class: "#{attribs[:class] || ''} button is-muted is-outlined js-markdown-tool",
                    data_action: action,
+                   draggable: false,
```

Comment:
Forgot to mention it, but this change solves a minor issue of being able to drag & drop markdown buttons themselves into the editor (resulting in `javascript:void(0)` being inserted)